src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1859,6 +1859,25 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 int
 Mavlink::task_main(int argc, char *argv[])
 {
+
```

Comment:
Can you add a comment here (needed for usb mavlink + console disabled)?

An alternative would be to make sure the fd's are reserved on the work queue, but this is fine here too.

Reply:
Hi @bkueng, thank you for input.
Sure thing, I have pushed some comments now.